src/main/java/io/cdap/plugin/gcp/gcs/actions/SourceDestConfig.java

```diff
@@ -74,15 +75,25 @@ public class SourceDestConfig extends GCPConfig {
     " at https://cloud.google.com/data-fusion/docs/how-to/customer-managed-encryption-keys")
   protected String cmekKey;
 
+  @Name(READ_TIMEOUT)
+  @Macro
+  @Nullable
+  @Description("Timeout in seconds to read data from an established HTTP connection (Default value is 20). " +
+    ("For performing copy/move operation on large files in GCS buckets, set a higher value. " +
+      "Set it to 0 for infinite(no limit)"))
+  protected String readTimeout;
```

Comment:
If the widget type is number, we can directly read it as Long saves an extra type conversion step.

Reply:
Set the type to Integer